provider/core/src/hello_world.rs

```diff
@@ -108,7 +108,7 @@ impl HelloWorldProvider {
         ("de-AT", "Servus Welt"),
         ("el", "Καλημέρα κόσμε"),
         ("en", "Hello World"),
-        ("en-001", "Hello from 🗺️"),           // WORLD
+        ("en-001", "Hello from 🗺️"),            // WORLD
```

Comment:
Lol fmt indeed.
The reason is that they are using the East_Asian_Width property, which is different between U+1F5FA and U+1F30D, because the latter has default emoji presentation and the former does not. The Variation Selector-16 following U+1F5FA should be taken into account and widen it.
To be fair, this is not really a solved problem. Hopefully the [Text Terminal Working Group of the UTC](https://www.unicode.org/consortium/techcommittees.html) (see [175-C12](https://www.unicode.org/L2/L2023/23076.htm#175-C12)) will come up with a specification for a solution.